docs-new/docs/graphql/core/auth/authentication/jwt.mdx

```diff
@@ -368,9 +357,7 @@ or
 }
 ```
 
-<div class="admonition">
-
-Important!
+:::info Important!
```

Comment:
Please make this `:::danger Important!` as this talks about security vulnerability